android/roc-android/build.gradle

```diff
@@ -105,50 +104,62 @@ artifacts {
-        name = project.name
-        userOrg = bintrayRepoOrg
-        configurations = ['archives']
-        desc = 'Android library for Roc Toolkit'
-        websiteUrl = 'https://github.com/roc-streaming/roc-java'
-        issueTrackerUrl = 'https://github.com/roc-streaming/roc-java/issues'
-        vcsUrl = 'https://github.com/roc-streaming/roc-java.git'
-        licenses = ["MIT"]
-        publish = true
-        publicDownloadNumbers = true
-
-        version {
-            gpg {
-                sign = true
-            }
-        }
+nexusStaging {
+    serverUrl = 'https://s01.oss.sonatype.org/service/local/'
+    packageGroup = 'org.roc-streaming'
+    username = ossrhUsername
+    password = ossrhPassword
+}
+
+signing {
+    if (signingKeyId?.trim() && signingKey?.trim() && signingPassword?.trim()) {
+        useInMemoryPgpKeys(signingKeyId, signingKey, signingPassword)
+        sign configurations.archives
     }
 }
 
-install {
-    repositories.mavenInstaller {
-        pom {
-            project {
+uploadArchives {
+    repositories {
+        mavenDeployer {
+            beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+            repository(url: "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/") {
+                authentication(userName: ossrhUsername, password: ossrhPassword)
+            }
+
+            snapshotRepository(url: "https://s01.oss.sonatype.org/content/repositories/snapshots/") {
+                authentication(userName: ossrhUsername, password: ossrhPassword)
+            }
+
+            pom.project {
+                name project.name
                 packaging 'aar'
-                name 'roc-android'
                 description 'Android library for Roc Toolkit'
                 url 'https://github.com/roc-streaming/roc-java'
 
+                scm {
+                    connection 'scm:git:git://github.com/roc-streaming/roc-java.git'
+                    developerConnection 'scm:git:ssh://github.com:roc-streaming/roc-java.git'
+                    url 'https://github.com/roc-streaming/roc-java'
+                }
+
                 licenses {
                     license {
                         name 'MIT'
                     }
                 }
 
-                scm {
-                    connection 'scm:git:https://github.com/roc-streaming/roc-java.git'
-                    developerConnection 'scm:git:https://github.com/roc-streaming/roc-java.git'
-                    url 'https://github.com/roc-streaming/roc-java'
+                developers {
+                    developer {
+                        id 'abushmin'
+                        name 'Andrey Bushmin'
+                        email 'diskbu@yandex.ru'
+                    }
+                    developer {
+                        id 'gavv'
+                        name 'Victor Gaydov'
+                        email 'victor@enise.org'
+                    }
                 }
```

Comment:
@MatteoArella Would you like to be added as well? (I guess then you need to be registered on maven central? Not sure.)

Reply:
Added Matteo.
@MatteoArella feel free to edit :)

Reply:
> I guess then you need to be registered on maven central?

No need